unityfiles/Assets/Scripts/Helper/Util/WUInteger.cs

```diff
@@ -0,0 +1,16 @@
+
+
+static class WUInteger
+{
```

Comment:
Maby change class name to something more readable

Reply:
Dette er Trygve sin klasse. Vi må ta en prat om naming her